litert/vendors/qualcomm/qnn_backend_test/test_utils.cc

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Qualcomm Innovation Center, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "litert/vendors/qualcomm/qnn_backend_test/test_utils.h"
+
+#include <sstream>
+#include <string_view>
+
+#include "litert/vendors/qualcomm/core/common.h"
+namespace litert::qnn {
+
+std::string QnnTestPrinter(
+    ::testing::TestParamInfo<std::tuple<::qnn::Options, std::string>>
+        param_info) {
```

Comment:
```suggestion
std::string QnnTestPrinter(
    const ::testing::TestParamInfo<std::tuple<::qnn::Options, std::string>>&
        param_info) {
```
Also, maybe std::string_view is enough?

Reply:
Thanks!